Project/Source/Modules/ModuleEditor.cpp

```diff
@@ -341,6 +344,17 @@ UpdateStatus ModuleEditor::Update() {
 		ImGui::EndPopup();
 	}
 
+	// CANT REMOVE COMPONENT MODAL
+	ImGui::SetNextWindowSize(ImVec2(400, 120), ImGuiCond_FirstUseEver);
+	if (ImGui::BeginPopupModal("Unable to remove component", nullptr, ImGuiWindowFlags_NoResize)) {
+		ImGui::PushTextWrapPos();
+		ImGui::Text("That component cannot be removed, there's some other component that requires its existance");
```

Comment:
"This Component cannot be removed. There are other Components in this Game Object that require its functionality."

Reply:
Quite right